core/src/test/java/io/parsingdata/metal/BitwiseValueExpressionSemanticsTest.java

```diff
@@ -41,7 +41,7 @@ public class BitwiseValueExpressionSemanticsTest extends ParameterizedParse {
 
     @Parameters(name = "{0} ({4})")
     public static Collection<Object[]> data() {
-        return Arrays.asList(new Object[][] {
+        return List.of(new Object[][] {
```

Comment:
I have been itching to convert these parameterized test using Stream<Arguments>, and get rid of Junit4 altogether, but that is a different issue. ;) Shall I create a new issue for this?

Reply:
Sounds like a good plan! :)